int8 id
float64 rssi
float64 m_rssi

